api.go

```diff
@@ -163,8 +169,11 @@ func handle(router *mux.Router, path string, methods []string, fn http.HandlerFu
 	wrapped := func(w http.ResponseWriter, r *http.Request) {
 		start := time.Now()
 		fn(w, r)
+		// Sanitize the URL string logging to prevent log message forgeries.
+		sanitizedURL := strings.ReplaceAll(r.URL.String(), "\n", "")
```

Comment:
https://go.dev/play/p/jBhUgjWNeTP shows that if you can get a newline in the raw query field, it will print a newline.